### ISSUE TITLE
Improve accessibility of herbmiller.me

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/written#23]
Improve the accessibility of herbmiller.me 
Well. I thought I'd done it. Both WAVE and axe Dev Tools on Chrome said it was OK

<img width="1896" height="1020" alt="Image" src="https://github.com/user-attachments/assets/857e208f-1677-4d46-950c-2b32c7cc4406" />

<img width="1900" height="950" alt="Image" src="https://github.com/user-attachments/assets/be60bf7c-a37d-4add-9f9c-20639b03ab2b" />

So why does Firefox say otherwise? 

<img width="1902" height="996" alt="Image" src="https://github.com/user-attachments/assets/59801a95-0717-4a57-a5f0-ef848040bbbf" />

**Comment by bobbingwide** [comment]
I've managed to reduce Errors and Color Contrast errors on my [Accessibility Statement](https://herbmiller.me/accessibility-statement/) but there are still quite a few alerts. 

<img width="1117" height="908" alt="Image" src="https://github.com/user-attachments/assets/40f1c98e-bbc3-48cc-a2d4-fe4ea101288b" />

There are 5 alerts on the Sharing buttons produced by Jetpack's Sharedaddy code. 
I may as well not display these sharing buttons, for pages at least.

**Comment by bobbingwide** [comment]
Jetpack says I should be using the block anyway.
But in the mean time I'll just disable the sharing buttons.

<img width="1299" height="350" alt="Image" src="https://github.com/user-attachments/assets/49bd3b4b-c74c-4133-955d-feb07c404a27" />

**Comment by bobbingwide** [comment]
The final alert on the Accessibility Statement page was produced by WooCommerce.
It contains this. 

```
<noscript><style>.woocommerce-product-gallery{ opacity: 1 !important; }</style></noscript>
```
WooCommerce produces this even when there is no product gallery being displayed.
